wrappers/wrapper.rb

```diff
@@ -501,11 +501,29 @@ def build_rest(rest, vehicle = nil)
       }
     end
 
+    def build_skills(job)
```

Comment:
build_skill_detail ? otherwise, it doesn't sound related to output/detail

Reply:
we already have _build_rest_, _build_timewindows_ ... called within _build_details_.